android/src/main/java/com/tanguyantoine/react/MusicControlModule.java

```diff
@@ -107,8 +108,8 @@ public void init() {
         md = new MediaMetadataCompat.Builder();
         pb = new PlaybackStateCompat.Builder();
         pb.setActions(controls);
-        nb = new NotificationCompat.Builder(context);
-        nb.setStyle(new NotificationCompat.MediaStyle().setMediaSession(session.getSessionToken()));
+        nb = new android.support.v4.app.NotificationCompat.Builder(context);
```

Comment:
Can you use imports instead of declaring the package name directly?

Reply:
There already imports